dotnetclr/datadog_checks/dotnetclr/data/conf.yaml.example

```diff
@@ -1,20 +1,50 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param host - string - required
     ## Host to monitor. "." means the current host,
     ## any other value makes the Agent attempt to connect to a remote host.
-    ## Note: Remote access requires additional permissions:
-    ## https://support.microsoft.com/en-us/help/922775/how-to-troubleshoot-monitoring-and-logging-issues-for-performance-coun
```

Comment:
this url does not exist anymore

Reply:
I wasn't able to access it on archive.org either…

Does this link qualify as a possible replacement? https://support.microsoft.com/en-us/help/969639/error-message-when-you-try-to-access-the-performance-monitor-perfmon-e

Reply:
I stumbled upon it, I don't know tbh

Reply:
Perhaps its more worthwhile to add remote monitoring steps in the README?